async-await.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+import {} from 'babel/polyfill';
+import fs from 'fs';
+import path from 'path';
+
+const dir = process.argv[2] || 'input';
+
+// fs module polyfill: this will be unnecessary if fs supports async/await
+function read(filePath) {
+  return new Promise(function(resolve, reject){
+    fs.readFile(filePath, {encoding: 'utf8'}, function(err, result){
+      if (err) { return reject(err); }
+      resolve(result.toString());
+    });
+  });
+}
```

Comment:
Let's copy the definition of [`readFile`][1] from __promises.js__. By avoiding cosmetic differences between the implementations we'll make the significant differences more apparent.


[1]: https://github.com/plaid/async-problem/blob/dfa1677fb196f5ac3d066cf643efdb712e1a0806/promises.js#L7-L17